### PR TITLE
Backport verify service accounts are set to stable 5

### PR DESCRIPTION
**Pull request by butonic**
backport https://github.com/owncloud/ocis/pull/8546 to stable 5

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -65,7 +65,11 @@
                 "GATEWAY_STORAGE_USERS_MOUNT_ID": "storage-users-1",
                 "STORAGE_USERS_MOUNT_ID": "storage-users-1",
                 // graph application ID
-                "GRAPH_APPLICATION_ID": "application-1"
+                "GRAPH_APPLICATION_ID": "application-1",
+
+                // service accounts
+                "OCIS_SERVICE_ACCOUNT_ID": "service-account-id",
+                "OCIS_SERVICE_ACCOUNT_SECRET": "service-account-secret"
```
Comment (kobergj) [inline .vscode/launch.json:72]: secrets should not be pushed to github

File: ocis-pkg/shared/errors.go [hunk 2]
```diff
@@ -69,3 +69,19 @@ func MissingAdminUserID(service string) error {
 		"the config/corresponding environment variable).",
 		service, defaults.BaseConfigPath())
 }
+
+func MissingServiceAccountID(service string) error {
```
Comment (kobergj) [inline ocis-pkg/shared/errors.go:73]: exported func `MissingServiceAccountID` should have a comment or be unexported

File: ocis-pkg/shared/errors.go [hunk 3]
```diff
@@ -69,3 +69,19 @@ func MissingAdminUserID(service string) error {
...
+	return fmt.Errorf("The service account id has not been configured for %s. "+
+		"Make sure your %s config contains the proper values "+
+		"(e.g. by running ocis init or setting it manually in "+
+		"the config/corresponding environment variable).",
+		service, defaults.BaseConfigPath())
+}
+
+func MissingServiceAccountSecret(service string) error {
```
Comment (kobergj) [inline ocis-pkg/shared/errors.go:81]: exported func `MissingServiceAccountSecret` should have a comment or be unexported

File: ocis/pkg/init/init.go [hunk 4]
```diff
@@ -83,6 +83,10 @@ type FrontendService struct {
 	ServiceAccount ServiceAccount `yaml:"service_account"`
 }
 
+type OcmService struct {
```
Comment (kobergj) [inline ocis/pkg/init/init.go:86]: exported type `OcmService` should have a comment or be unexported

**Review by kobergj — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (butonic) [reply .vscode/launch.json:72]: these ar secrets for the launch.json that is used by developers to just run an ocis instance. The file sets all necessary env vars to start ocis withour running ocis init.

File: ocis-pkg/shared/errors.go (hunk 2 above)
Comment (butonic) [reply ocis-pkg/shared/errors.go:73]: this is a backport and we should not add something that is not in master

File: ocis-pkg/shared/errors.go (hunk 3 above)
Comment (butonic) [reply ocis-pkg/shared/errors.go:81]: this is a backport and we should not add something that is not in master

File: ocis/pkg/init/init.go (hunk 4 above)
Comment (butonic) [reply ocis/pkg/init/init.go:86]: this is a backport and we should not add something that is not in master

**Comment by micbar** [comment]
@butonic @kobergj why is our linter not configured correctly to spot that missing code comment right away?

Quick win imho.

**Review by micbar — APPROVED** [review]

**Comment by rhafer** [comment]
This is now part of #8627

**Pull request closed without merging**